setup.cfg

```diff
@@ -1,11 +1,11 @@
 [metadata]
 name = ansible-runner
 author = Ansible, Inc.
-author-email = info@ansible.com
+author_email = info@ansible.com
```

Comment:
I'm not sure if this is right. It's not a setuptools option, it's coming from `pbr`: https://docs.openstack.org/pbr/latest/user/using.html#setup-cfg.

If you want to use native setuptools settings, see https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html#metadata / https://packaging.python.org/guides/distributing-packages-using-setuptools/

FWIW I'd split this change into a separate PR